docker-compose.yml

```diff
@@ -150,6 +151,26 @@ services:
     networks:
       - elasticsearch
 
+  ingest-test:
+    profiles:
+      - dev
```

Comment:
Do we need dev?

Reply:
I guess that's a leftover, I'll remove it